.changeset/thin-apricots-help.md

```diff
@@ -0,0 +1,6 @@
+---
+'@lit/localize-tools': minor
```

Comment:
No action required – just curious: Why `minor` and not `patch`?

Reply:
Yeah I think patch makes sense for this too. `@lit/localize-tools` is at 0.6.9. I wouldn't consider this a breaking change.

```suggestion
'@lit/localize-tools': patch
```

Reply:
wasn't too sure tbh. i figured both should be a patch but wasn't sure what we do when we bump a major version of a dependency.

ill update